### PR TITLE
Coconut Lime Cilantro Cooler

### PR DESCRIPTION
**Pull request by FionaFR**
Coconut Lime Cilantro Cooler 
Recipe and Picture

### CODE REVIEW

File: src/recipes/coconut-lime-cilantro-cooler.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            "ingredient": "lime for garnish"
+        }
+        
+    ],
+    "directions": [
+        "Step 1: In a glass, stir all ingredients together except for the two garnishes.",
+        "Step 2: Garnish with the slice of lime and the 2 fresh cilantro leaves",
+        "Step 3: Serve immediately."
+    ],
+    "image": "coconut-lime-cilantro-cooler.jpg>",
```
Comment (alfg) [inline src/recipes/coconut-lime-cilantro-cooler.json:53]:
```suggestion
    "image": "coconut-lime-cilantro-cooler.jpg",

```

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, added a suggestion to load the image file.

**Review by FionaFR — COMMENTED** [review]
Thanks. Changes were made

File: src/recipes/coconut-lime-cilantro-cooler.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            "ingredient": "rum(optional)"
+        },
+        {
+            "quantity": "1",
+            "measure": "teaspoon",
+            "ingredient": "fresh cilantro chopped"
+        },
+        {
+            "quantity": "3",
+            "measure": "cubes>",
```
Comment (alfg) [inline src/recipes/coconut-lime-cilantro-cooler.json:33]:
```suggestion
            "measure": "cubes",

```

File: src/recipes/coconut-lime-cilantro-cooler.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            "ingredient": "ice"
+        },
+        {
+            "quantity": "2",
+            "measure": "leaves",
+            "ingredient": "fresh cilantro for garnish"
+        },
+        {
+            "quantity": "1",
+            "measure": "slice>",
```
Comment (alfg) [inline src/recipes/coconut-lime-cilantro-cooler.json:43]:
```suggestion
            "measure": "slice",

```

File: src/recipes/coconut-lime-cilantro-cooler.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            "ingredient": "lime for garnish"
+        }
+        
+    ],
+    "directions": [
+        "Step 1: In a glass, stir all ingredients together except for the two garnishes.",
+        "Step 2: Garnish with the slice of lime and the 2 fresh cilantro leaves",
+        "Step 3: Serve immediately."
```
Comment (alfg) [inline src/recipes/coconut-lime-cilantro-cooler.json:51]: Can you remove the Steps numbers? These will be rendered as an ordered list.
```suggestion
        "In a glass, stir all ingredients together except for the two garnishes.",
        "Garnish with the slice of lime and the 2 fresh cilantro leaves",
        "Serve immediately."

```

**Review by alfg — CHANGES_REQUESTED** [review]
Hey, sorry, added a few more suggestions before we can merge. Thanks!

**Review by FionaFR — COMMENTED** [review]
The three changes were made. Thanks

**Comment by alfg** [comment]
Great! Thanks for fixing!

**Comment by FionaFR** [comment]
You're welcome :)

**Pull request merged**